docs/content/howto/dataframe-api.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Get data out from Rerun with code
+order: 1600
+---
+
+Rerun 0.19 added the Dataframe API to its SDK, with enables getting data out of Rerun from code. This page provides an overview of the API, as well as recipes to load the data in popular packages such as [Pandas](https://pandas.pydata.org), [Polars](https://pola.rs), and [DuckDB](https://duckdb.org).
+
+<!-- TODO(#7499): add links to the Python SDK documentation where appropriate -->
+
+## The dataframe API
+
+### Loading a recording
+
+A recording can be loaded from a RRD using the `load_recording()` function:
+
+```python
+import rerun as rr
+
+recording = rr.dataframe.load_recording("/path/to/file.rrd")
+```
+
+Although RRD files generally contain a single recording, they may occasionally contain 2 or more. This can happen, for example, if the RRD includes a blueprint, which is stored as a recording that is separate from the data.
+
+For such RRD, the `load_archive()` function can be used:
+
+
+<!-- NOLINT_START -->
+```python
+import rerun as rr
+
+archive = rr.dataframe.load_archive("/pat/to/file.rrd")
+
+print(f"The archive contains {archive.num_recordings()} recordings.")
+
+for recording in archive.all_recordings():
+    ...
+```
+<!-- NOLINT_END -->
+
+The overall content of the recording can be inspected using the `schema()` method:
+
+```python
+schema = recording.schema()
+schema.index_columns()        # list of all index columns (timelines)
+schema.component_columns()    # list of all component columns
+```
+
+
+### Creating a view
+
+The first step for getting data out of a recording is to create a view, which requires specifying an index column and some content to include.
```

Comment:
super nit: "some content" is a bit vague, perhaps lose the "some" or be bit more specific.